src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -90,23 +95,54 @@ class HeaderHelpMenuUI extends Component<Props, State> {
 
     const defaultContent = useDefaultContent ? (
       <Fragment>
-        <EuiText size="s">
-          <p>
-            <FormattedMessage
-              id="core.ui.chrome.headerGlobalNav.helpMenuHelpDescription"
-              defaultMessage="Get updates, information, and answers in our documentation."
-            />
-          </p>
-        </EuiText>
-
-        <EuiSpacer />
-
-        <EuiButton iconType="popout" href={kibanaDocLink} target="_blank">
+        <EuiButtonEmpty href={kibanaDocLink} target="_blank" size="xs">
```

Comment:
All these EuiButtonEmpty's look great. To make them line up with the popover title, though, can you add `flush="left"` to all of them?

Reply:
Good catch, I made the fix here: https://github.com/elastic/kibana/pull/49797/commits/22d59cb3662184f2acc3dcc314973f7bb04f83a5. Updated screenshot can be found here: https://github.com/elastic/kibana/pull/49797#issuecomment-548913726.